app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    post.update(commentscounter: post.comments.all.length)
```

Comment:
Kindly consider following the [ruby best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) by using the correct naming convention here. Instead of `commentscounter`, you can use the snake_case format `comments_counter` to improve your code and make it more standard. Remember, using standard naming conventions in Rails helps in fostering consistency, leveraging Rails' automagic features, improving collaboration, and making your codebase more readable and maintainable. It's a foundational practice that contributes to the overall success of your Rails projects.